packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js

```diff
@@ -9,108 +9,78 @@ describe('datetime-picker/Value', () => {
-      expect(el.inputEl.value).to.be.equal('01-Apr-2020', 'Input value has not updated');
+      expect(inputElement(el).value).to.be.equal('2020-04-01', 'Input value has not updated');
     });
     it('It should be possible to select value in range duplex mode', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" opened range duplex></ef-datetime-picker>');
       el.views = ['2020-04', '2020-05'];
       await elementUpdated(el);
-      await nextFrame();
-      await nextFrame();
+      await nextFrame(el);
+      await nextFrame(el);
 
-      const calendarEl = el.calendarEl;
+      const calendarEl = calendarElement(el);
       const fromCell = calendarEl.shadowRoot.querySelectorAll('div[tabindex]')[0]; // 2020-04-01
       fromCell.click();
       await elementUpdated(el);
-      await nextFrame();
+      await nextFrame(el);
 
-      const calendarToEl = el.calendarToEl;
+      const calendarToEl = calendarToElement(el);
       const toCell = calendarToEl.shadowRoot.querySelectorAll('div[tabindex]')[0]; // 2020-05-01
       toCell.click();
       await elementUpdated(el);
-      await nextFrame();
+      await nextFrame(el);
 
       expect(el.values[0]).to.be.equal('2020-04-01', 'Value from has not been updated');
       expect(el.values[1]).to.be.equal('2020-05-01', 'Value to has not been update');
 
-      expect(el.inputEl.value).to.be.equal('01-Apr-2020', 'Input from value has not updated');
-      expect(el.inputToEl.value).to.be.equal('01-May-2020', 'Input to value has not updated');
+      expect(inputElement(el).value).to.be.equal('2020-04-01', 'Input from value has not updated');
+      expect(inputToElement(el).value).to.be.equal('2020-05-01', 'Input to value has not updated');
     });
     it('Timepicker value is populated', async () => {
-      const el = await fixture('<ef-datetime-picker lang="en-gb" opened timepicker with-seconds value="2020-04-21T13:14:15"></ef-datetime-picker>');
-      const timePicker = el.timepickerEl;
+      const el = await fixture('<ef-datetime-picker lang="en-gb" opened timepicker show-seconds value="2020-04-21T13:14:15"></ef-datetime-picker>');
+      const timePicker = timePickerElement(el);
       expect(timePicker.hours).to.equal(13);
       expect(timePicker.minutes).to.equal(14);
       expect(timePicker.seconds).to.equal(15);
     });
     it('It should be possible to change timepicker value', async () => {
-      const el = await fixture('<ef-datetime-picker lang="en-gb" opened timepicker with-seconds value="2020-04-21T13:14:15"></ef-datetime-picker>');
-      const timePicker = el.timepickerEl;
-      typeText(timePicker, '16:17:18');
+      const el = await fixture('<ef-datetime-picker lang="en-gb" opened timepicker show-seconds value="2020-04-21T13:14:15"></ef-datetime-picker>');
+      typeText(timePickerElement(el), '16:17:18');
       expect(el.value).to.equal('2020-04-21T16:17:18');
     });
```

Comment:
Add a test, `it should be possible to change formatOptions value` and `it should be possible to change locale value`

Reply:
Added.